### DISCUSSION TITLE
Carbon ads should go on the right sidebar

### DISCUSSION THREAD

**Discussion by okandas** [TanStack/query#5778] [General] [answered]
![image](https://github.com/TanStack/query/assets/26438624/46036f21-f153-47dd-acf6-7234cbc17059).

Small fix, is it possible to place carbon ads on the right sidebar so that they do not interfere  with the left sidebar navigation ..

**Comment by TkDodo** [comment, marked as answer]
this comes from the https://github.com/tanstack/tanstack.com repo

**Reply by okandas** [reply]
@TkDodo here you go - https://github.com/TanStack/tanstack.com/pull/91
